django/contrib/admin/options.py

```diff
@@ -250,7 +250,7 @@ def formfield_for_choice_field(self, db_field, request, **kwargs):
     def get_field_queryset(self, db, db_field, request):
         """
         If the ModelAdmin specifies ordering, the queryset should respect that
-        ordering.  Otherwise don't specify the queryset, let the field decide
+        ordering. Otherwise don't specify the queryset, let the field decide
```

Comment:
I think we usually try to avoid unrelated cleanup in PRs but I'll let the fellows be the judge, I don't really know.